include/astshim/FitsChan.h

```diff
@@ -485,6 +487,14 @@ class FitsChan : public Channel {
     std::string getCardName() const { return getC("CardName"); }
 
     /**
+    Get the name of all cards, in order, starting from the first card
+
+    Not "const" because the code changes the index of the current card while operating
+    (but restores the original index when done).
+    */
+    std::vector<std::string> getCardNames();
```

Comment:
If astshim is going into AST, then shouldn't we have our own wrappers that insulate us from changes in that?  All of this seems to be in this strange no man's land, not quite LSST, but not quite AST.  We're exposing far more of AST than we do with any other package, including Eigen.